content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -0,0 +1,325 @@
+			</td>
+		</tr>
+		<tr>
+			<td>Privilege Escalation</td>
+			<td>Privilege escalation to root should not be allowed.
+				<p>Restricted Fields:</p>
+				<ul>
+					<li><code>spec.containers[*].securityContext.privileged</code></li>
+					<li><code>spec.initContainers[*].securityContext.privileged</code></li>
+				</ul>
+				<p>Allowed Values: <code>false</code>, undefined/nil</p>
+			</td>
+		</tr>
+		<tr>
+			<td>Running as Non-root</td>
+			<td>Containers must be required to run as non-root users.
+				<p>Restricted Fields:</p>
+				<ul>
+					<li><code>spec.securityContext.runAsNonRoot</code></li>
+					<li><code>spec.containers[*].securityContext.runAsNonRoot</code></li>
+					<li><code>spec.initContainers[*].securityContext.runAsNonRoot</code></li>
+				</ul>
+				<p>Allowed Values: <code>true</code></p>
+			</td>
+		</tr>
+		<tr>
+			<td>Non-root groups <em>(optional)</em></td>
+			<td>Containers should be forbidden from running with a root primary or supplementary GID.
+				<p>Restricted Fields:</p>
+				<ul>
+					<li><code>spec.securityContext.runAsGroup</code></li>
+					<li><code>spec.securityContext.supplementalGroups[*]</code></li>
+					<li><code>spec.securityContext.fsGroup</code></li>
+					<li><code>spec.containers[*].securityContext.runAsGroup</code></li>
+					<li><code>spec.containers[*].securityContext.supplementalGroups[*]</code></li>
+					<li><code>spec.containers[*].securityContext.fsGroup</code></li>
+					<li><code>spec.initContainers[*].securityContext.runAsGroup</code></li>
+					<li><code>spec.initContainers[*].securityContext.supplementalGroups[*]</code></li>
+					<li><code>spec.initContainers[*].securityContext.fsGroup</code></li>
+				</ul>
+				<p>Allowed Values:</p>
+				<ul>
+					<li><code>non-zero</code></li>
+					<li>undefined / nil (except for <code>*.runAsGroup</code>)</li>
+				</ul>
+			</td>
+		</tr>
+		<tr>
+			<td>Seccomp</td>
+			<td>
+				<p>The <code>runtime/default</code> seccomp profile must be required, or allow additional whitelisted values.</p>
+				<p>Restricted Fields:</p>
+				<ul>
+					<li><code>metadata.annotations['seccomp.security.alpha.kubernetes.io/pod']</code></li>
+					<li><code>metadata.annotations['container.seccomp.security.alpha.kubernetes.io/*']</code></li>
+				</ul>
+				<p>Allowed Values:</p>
+				<ul>
+					<li><code>runtime/default</code></li>
+					<li>undefined (container annotation)</li>
+				</ul>
+			</td>
+		</tr>
+	</tbody>
+</table>
+
+## Policy Instantiation
+
+Decoupling policy definition from policy instantiation allows for a common understanding and
+consistent language of policies across clusters, independent of the underlying enforcement
+mechanism.
+
+As mechanisms mature, they will be defined below on a per-policy basis. The methods of enforcement
+of individual policies are not defined here.
+
+**PodSecurityPolicy**
+
+*   [Privileged](https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/policy/privileged-psp.yaml)
```

Comment:
nit: You can remove the extra spaces at start of list item text (lines 274-276)

Reply:
Done. (It was an artifact of the automatic conversion from a google doc to markdown).